components/Sidebar.tsx

```diff
@@ -70,83 +97,188 @@ const Sidebar: React.FC<Props> = ({ menuState, nav: currentNav }) => {
+                        <Text variant="sm" css={{ color: '$primaryLight' }}>
+                            Back to Home
+                        </Text>
+                    </Flex>
+
+                    {/* Sidebar Version Section */}
+                    {showPlatformSelector ? (
+                        <section
+                            style={{
+                                margin: '0px 0.5rem 0.5rem 0.4rem',
+                                position: 'sticky',
+                                top: '0',
+                                zIndex: '20',
+                                background: 'var(--docs_bg_content)'
+                            }}>
+                            <Listbox value={tech} onChange={changeTech}>
+                                <Listbox.Button className="dropdown">
+                                    <div style={{ display: 'flex ', alignItems: 'center' }}>
+                                        {tech.icon}
+                                        <span style={{ marginLeft: '1rem' }}>{tech.name}</span>
+                                    </div>
+                                    <ChevronDownIcon />
+                                </Listbox.Button>
+                                <Listbox.Options className="dropdown-options">
+                                    {menuItem.map((m) => (
+                                        <Listbox.Option
+                                            key={m.link}
+                                            value={m}
+                                            className={({ active }) =>
+                                                `${
+                                                    active
+                                                        ? 'dropdown-option dropdown-option-active'
+                                                        : 'dropdown-option'
+                                                }`
+                                            }>
+                                            {m.icon}
+                                            <span style={{ marginLeft: '1rem' }}>{m.name}</span>
+                                        </Listbox.Option>
+                                    ))}
+                                </Listbox.Options>
+                            </Listbox>
+                        </section>
+                    ) : null}
+                    {/* Sidebar Menu Section */}
+                    {nav
+                        ? Object.entries(nav).map(([key, children], index) => (
+                              <SidebarSection key={key} value={key} index={index} nested={false}>
+                                  {children as React.ReactChildren}
+                              </SidebarSection>
+                          ))
+                        : null}
+                </div>
+            ) : null}
             <style jsx>{`
                 .sidebar {
```

Comment:
<img width="449" alt="image" src="https://user-images.githubusercontent.com/57426646/216512341-fb58c563-d873-4c23-b8ca-55394d7fb40f.png">

Allows more control using state variables. Not using inline CSS since these styles are used in multiple divs, created classes instead

Reply:
dont write inside style tag 

also dont get why we cant use react-ui components for the same and use css there 
this is just making code unreadable